### DISCUSSION TITLE
where is lin-x86_64

### DISCUSSION THREAD

**Discussion by yoyostudy** [surgical-robotics-ai/surgical_robotics_challenge#87] [Q&A]
As suggested, we need to run the simulation in folder `ambf/bin/lin-x86_64`. Unfortunately, I only found `mac-i386`  `mac-x86_64`  `win-Win32`  `win-x64` under the folder `ambf/bin`.

Is there anything wrong with my setup of ambf?

Thank you so much

**Comment by adnanmunawar** [comment]
Hello, did you build AMBF and was it successful?
